src/main/java/com/celements/web/plugin/cmd/ExternalJavaScriptFilesCommand.java

```diff
@@ -291,14 +291,13 @@ void injectDisplayAll(boolean displayedAll) {
   }
 
   String getExtStringForJsFile(JsFileEntry jsFile) {
-    return "<script" + ((jsFile.getLoadMode() != JsLoadMode.SYNC)
-        ? " " + jsFile.getLoadMode().toString().toLowerCase()
-        : "") + " type=\""
-        + (jsFile.isModule()
-            ? "module"
-            : "text/javascript")
-        + "\" src=\""
-        + StringEscapeUtils.escapeHtml(jsFile.getFilepath()) + "\"></script>";
+    var loadMode = Optional.ofNullable(jsFile.getLoadMode())
+        .filter(mode -> !jsFile.isModule() && (mode != JsLoadMode.SYNC));
```

Comment:
This is wrongly filtering loadMode=async for module which is valid and therfore must be supported. See [https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script)

-> obviously a unit test for type=module and loadMode=async is missing :-) therefore please add one.

Reply:
this cannot happen because of the implementation of `JsFileEntry.getLoadMode()`
```
  public JsLoadMode getLoadMode() {
    if (isModule()) {
      return JsLoadMode.DEFER;
    }
    return loadMode;
  }
```

Reply:
I've changed the check to `(isModule() && (loadMode == SYNC))`